pkgs/development/cuda-modules/aliases.nix

```diff
@@ -15,4 +14,8 @@ in
   autoAddOpenGLRunpathHook =
     mkRenamed "autoAddOpenGLRunpathHook" "pkgs.autoAddDriverRunpath"
       final.pkgs.autoAddDriverRunpath; # Added 2024-03-30
+  markForCudatoolkitRootHook =
+    mkRenamed "markForCudatoolkitRootHook" "cudaPackages.markForCudatoolkitRoot"
+      final.markForCudatoolkitRoot; # Added 2024-04-19
+  setupCudaHook = mkRenamed "setupCudaHook" "cudaPackages.setupCuda" final.setupCuda; # Added 2024-04-19
```

Comment:
I remember a discussion about this, and I agree there's little point in having the suffix, and it's ok not to have it... I don't know why bother renaming old stuff, but no strong opinions here on my part